doc/translate.txt

```diff
@@ -43,3 +44,17 @@ Quit the translation buffer
   :TranslateClear
 <
 See https://github.com/soimort/translate-shell#usage for more.
+
+
+==============================================================================
+2. Configuration                                             *translate-usage*
+
+2.1 g:translate_default_lang                        *g:translate_default_lang*
+
+A dictionary of defaults source → target.
+>
+  let g:translate_default_lang = {
+        \ 'fr': 'en',
+        \ 'en': 'fr'
```

Comment:
Instead of setting both way, what about to do the opposite implicitly ? If you set `fr: en`, then automatically it will try the reverse also `en: fr`

Reply:
Yeah, I thought about it... but didn't know if then I should use a dict, list etc...
Also it would need to be mentioned in the doc that it translates in both directions..
so I preferred to keep it as simple as possible..

Reply:
Well, the simplest way for the user is to set a string instead of a dict, but if this does the job then okey

Reply:
But using the string `en:fr` is a bit confusing as it means `en -> fr` in translate-shell

Also we can't  have 2 variables `default_source`, `default_target` because the idea is that the `source` could be the `target` 
What do you think ?

Reply:
And what about a List of favorite language ? `['fr', 'en']` called `g:favorite_lang` ?